### DISCUSSION TITLE
Where to find the sample application?

### DISCUSSION THREAD

**Discussion by AlphabetsAlphabets** [dotnet-bluetooth-le/dotnet-bluetooth-le#900] [Q&A]
As the title says, where can I find it? I've looked through the Source folder but nothing and the README states there is a sample app. Am I missing something?

**Comment by janusw** [comment]
All the sample-client apps can be found here:
https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le/tree/master/Source/BLE.Client

Sorry if that is not obvious, I'll update the README acoordingly.

There are several samples by now, the most important one at this point is probably the one for MAUI:
* https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le/tree/master/Source/BLE.Client/BLE.Client.Maui

Alternatively, there is one for the Windows Console:
* https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le/tree/master/Source/BLE.Client/BLE.Client.WinConsole

And there is also still the old Xamarin client:
* https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le/tree/master/Source/BLE.Client/BLE.Client
* https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le/tree/master/Source/BLE.Client/BLE.Client.Droid
* https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le/tree/master/Source/BLE.Client/BLE.Client.iOS
* https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le/tree/master/Source/BLE.Client/BLE.Client.UWP
* https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le/tree/master/Source/BLE.Client/BLE.Client.macOS

HTH!

**Comment by AlphabetsAlphabets** [comment]
Hello, thanks for the help! Yeah, it wasn't that obvious. I looked at the Test folder thinking it would be there. Didn't think it would be with the source code.

Sent from Proton Mail Android

-------- Original Message --------

**Comment by AlphabetsAlphabets** [comment]
Hi, @janusw I was able to load up the sample but I'm not able to run it. I haven't been able to find the build errors either. 
![image](https://github.com/user-attachments/assets/6f4dbae5-dc28-4c55-a9ce-6c4606d7c08e)

I tried to search for .NET 5.0 on the website listed in the error but I can't find it. I'm not sure how to run the sample. Sorry if it's an obvious question. Perhaps it's the way I'm running the sample.
1. I cloned the entire project
2. Selected `BLE.Client.MAUI`
![image](https://github.com/user-attachments/assets/0923b99e-ed3a-43eb-8161-cab5b93657ca)
3. Swapped to the android device I have connected.
4. Clicked the play button.

The build progress bar appears then disappears and says "build failed."
